Dockerfile

```diff
@@ -11,11 +11,15 @@ ENV PATH="/opt/depot_tools:${PATH}"
 
 RUN mkdir /root/repo
 WORKDIR /root/repo
-ARG revision=pandafium
-RUN gclient config --unmanaged --name pdfium https://github.com/gradescope/pdfium.git@origin/$revision
-RUN gclient sync
+
+RUN gclient config --unmanaged --name pdfium https://github.com/gradescope/pdfium.git
+ADD . / pdfium/
 
 WORKDIR /root/repo/pdfium
+ARG revision=HEAD
+RUN git remote set-url origin https://github.com/gradescope/pdfium.git
```

Comment:
this is because if your local repo was checked out over SSH, you won't be able to fetch it inside the Docker container